src/index.ts

```diff
@@ -0,0 +1,57 @@
+import kthLog from '@kth/log'
+import mongoose, { type ConnectOptions } from 'mongoose'
+
+let isConnected = false
+
+// True if default connection to MongoDB is currently established
+export function isOk() {
+  return isConnected === true
```

Comment:
https://masteringjs.io/tutorials/mongoose/connection-status

Not sure if the link above is still relevant, but we might be able to use `mongoose.connection.readyState == 1` instead and drop `isConnected`